vlib/vweb/README.md

```diff
@@ -219,9 +219,34 @@ You have access to the raw request data such as headers
 or the request body by accessing `app` (which is `vweb.Context`).
 If you want to read the request body, you can do that by calling `app.req.data`.
 To read the request headers, you just call `app.req.header` and access the
-header you want, e.g. `app.req.header.get(.content_type)`. See `struct Header`
+header you want example. `app.req.header.get(.content_type)`. See `struct Header`
 for all available methods (`v doc net.http Header`).
+It has, too, fields for the `query`, `form`, `files`.
 
+#### - Query
+To handle the query context, you just need use the  `query` field
+
+**Example:**
+
+```v ignore
```

Comment:
Why `v ignore`, instead of just `v` ? 

The example seems simple, complete and thus easy to compile.

`v ignore` should be used for blocks that have incomplete/invalid code.